DeAceroDLabs/marketplace-frontend
# [DMP-450] Create user on Auth0

## What is done

[[DMP-450] Carga valores usando el código postal](https://deacero.atlassian.net/browse/DMP-450)

## How is done

1. Created a new api file for handling user related calls..
2. Created a new method for handling user creation on Auth0.
3. Added the credentials of the Auth0 app in .env file.

## How to test

1. Install new dependencies `npm i`
2. Run `npm start`.
3. Go to `/signup`.
4. Fill out the form and submit it.
5. You should see the success popup if the account was created successfully.
6. Go to [mkt-place app dashboard](https://manage.auth0.com/dashboard/us/mkt-place/) using the credentials shared via slack.
7. Go to `Try your Login box``
8. Use the credentials you used for creating your user. You should be redirected to a success page.
